examples/ComposeModules/index.tsx

```diff
@@ -0,0 +1,14 @@
+import * as ReactDOM from 'react-dom'
+
+import * as DebugHtml from '../../src/Debug/Html'
+import * as React from '../../src/React'
+
+import * as App from './App'
+
+const program = process.env.NODE_ENV === 'production' ? React.program : DebugHtml.programWithDebugger
```

Comment:
@IMax153 it's better to use `programWithFlags` instead of simple `program`:

```ts
const program = process.env.NODE_ENV === 'production' ?
  React.programWithFlags : 
  DebugHtml.programWithDebuggerWithFlags

const main = program(App.init, App.update, App.view)

React.run(main({prefix: 'String: '}), dom => {
  ReactDOM.render(dom, document.getElementById('root'))
})
```

Reply:
See https://github.com/gcanti/elm-ts/pull/44/commits/52ac6e3942d66ddd4741f606b485f202f4ab23ca